artemis/reporting/modules/drupal_scanner/template_old_drupal.jinja2

```diff
@@ -0,0 +1,21 @@
+{% if "old_drupal" in data.contains_type %}
+    <li>{% trans %}The following addresses contain old Drupal versions:{% endtrans %}
+        <ul>
+            {% for report in data.reports %}
+                {% if report.report_type == "old_drupal" %}
+                    <li>
+                        {{ report.target }} - Drupal {{ report.additional_data.version }}
+                        {{ report_meta(report) }}
+                    </li>
+                {% endif %}
+            {% endfor %}
+        </ul>
+        <p>
+            {% trans trimmed %}
+                If a site is no longer used, we recommend shutting it down to eliminate the risk
+                of exploitation of known vulnerabilities in older Drupal versions. Otherwise, we
+                recommend regular Drupal core and plugin updates.
```

Comment:
```suggestion
                recommend following regular Drupal core and plugin updates.
```

Reply:
why is this one better? IMO `update` has two meanings, "what you do on your server" and "what appears on drupal release list" and both are OK and what it's now in the repo is simpler